react/src/index.js

```diff
@@ -285,7 +289,36 @@ const App = () => {
   // Crasher parses query params sent by /tests for triggering crashes for Release Health
   crasher();
 
+
+  //LaunchDarkly settings
+  const [lDFeatureFlag, setLDFeatureFlag] = React.useState(false);
+  const lDUser = {
+    "kind": queryParams.get('se') === null || !queryParams.get('se').match("-tda-") ? "user": "automation",
```

Comment:
If the se tag is not empty and contains _-tda-_, e.g. prod-tda-direct--desktop_web/test_products_join.py-sauce-chrome_windows, then user > kind: automation
Otherwise, user > kind: "user"

Reply:
My idea is that if kind: automation, then it would be part of the [Experiment](https://docs.launchdarkly.com/home/experimentation); if kind: user, then it would not.